mqtt3/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,63 @@
+= MQTT 3 Connector 1.0
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for MQTT 3 (MQTT 3 Connector) is an MQTT (Message Queuing Telemetry Transport) v3.x protocol-compliant MuleSoft extension. MQTT is a lightweight publish/subscribe messaging protocol designed for connecting remote devices with a small code footprint and minimal network bandwidth. MQTT today is used in a wide variety of industries, such as automotive, manufacturing, telecommunications, and oil and gas.
+
+The connector consumes and produces MQTT messages and supports all MQTT v3.x functionalities including message retention, last will and testament messages, and persistent sessions.
+
+For compatibility information and fixed issues, see the MQTT 3 Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* MQTT v3.x protocol
+* Anypoint Connectors
+* Mule runtime engine (Mule)
```

Comment:
@emariotti3 can you confirm is we support Mule RT 3.0? @kevintroller , it may be useful to clarify Mule RT version.

Reply:
This is a Mule 4 connector. Mule 3 is not supported by any Mule 4 connectors. They are completely different technologies. The supported Mule runtime version is documented in release notes.